INSTALL.md

```diff
@@ -79,10 +79,11 @@ codiumai/pr-agent@v0.9
 git clone https://github.com/Codium-ai/pr-agent.git
 ```
 
-2. Install the requirements in your favorite virtual environment:
+2. Navigate to the `/pr-agent` folder and install the requirements in your favorite virtual environment:
 
+*Tip: make sure Rust is installed and in your `PATH`, instructions: https://rustup.rs*
 ```
-pip install -r requirements.txt
+pip install -e .
```

Comment:
can you explain this line ? hy do we need this change ?

Reply:
The original wouldn't install `pr_agent` as a `pip` package and thus couldn't be called by `python3`, one could call `python3 -m cli.py ...` but wouldn't have ability to call `pr_agent[.cli]` First ran into it by following the installation guide and it could not find `pr_agent`

The `-e` is not necessary, but it makes it possible to edit the source files without having to re-install the pip package every time

I need to add `requirements.txt` to `setup.py` though, will do that in a few minutes

Reply:
Just tested on a clean environment, `pip install .` will install the dependencies automatically, so no need to edit `setup.py`, but the `-e` flag is optional

Reply:
what was wrong with the old line ?